src/Lifecycle.ts

```diff
@@ -71,13 +71,22 @@ import { persistOidcAuthenticatedSettings } from "./utils/oidc/persistOidcSettin
 const HOMESERVER_URL_KEY = "mx_hs_url";
 const ID_SERVER_URL_KEY = "mx_is_url";
 
+/**
```

Comment:
`/**`-style comments only apply to the immediately following thing, so this leaves `REFRESH_TOKEN_STORAGE_KEY` without any documentation.

In this case I don't think  the tsdoc documentation is important (since these are private fields), so a regular comment is fine. But if we're going to rely on it being read as a regular comment, we shouldn't pretend it's a tsdoc comment:

```suggestion
/*
```

Likewise the others below.